server/controllers/templates/web_templates.go

```diff
@@ -352,6 +352,75 @@ v{{ .AtlantisVersion }}
 </html>
 `))
 
+// pullStreamData holds the data needed to stream the current PR information
+type LogStreamData struct {
+	AtlantisVersion string
+	PullInfo        string
+	CleanedBasePath string
+}
+
+var LogStreamingTemplate = template.Must(template.New("blank.html.tmpl").Parse(`
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>atlantis</title>
+    <meta name="description" content>
+    <meta name="author" content>
+    <link
+       rel="stylesheet"
+       href="https://cdnjs.cloudflare.com/ajax/libs/xterm/3.14.5/xterm.min.css"
+       integrity="sha256-uTIrmf95e6IHlacC0wpDaPS58eWF314UC7OgdrD6AdU="
+       crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="/static/css/normalize.css">
+    <link rel="stylesheet" href="/static/css/skeleton.css">
+    <link rel="stylesheet" href="/static/css/custom.css">
+    <link rel="icon" type="image/png" href="/static/images/atlantis-icon.png">
+    <style>
+      #terminal {
+        width: 100%;
+        height: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <section class="header">
+      <a title="atlantis" href="/"><img class="hero" src="/static/images/atlantis-icon_512.png"/></a>
+      <p class="title-heading">atlantis</p>
+      <p class="title-heading"><strong></strong></p>
+      </section>
+      <div class="spacer"></div>
+      <br>
+      <section>
+        <div id="terminal"></div>
+      </section>
+    </div>
+    <footer>
+    </footer>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
+    <script src="https://unpkg.com/xterm@4.9.0/lib/xterm.js"></script>
+    <script src="https://unpkg.com/xterm-addon-attach@0.6.0/lib/xterm-addon-attach.js"></script>
+    <script src="https://unpkg.com/xterm-addon-fit@0.4.0/lib/xterm-addon-fit.js"></script>
+
+    <script>
+      var term = new Terminal();
+      var socket = new WebSocket("wss://log_streaming_ws");
```

Comment:
This needs to be full url to the server. Upstream [example](https://github.com/runatlantis/atlantis/pull/1315/files#diff-98446d84bdd9f6860dfe6c4d3f011675dbe05898a57034cd04e84a2489793175R292). Something like this
```suggestion
      var socket = new WebSocket(
      (document.location.protocol === "http:" ? "ws://" : "wss://") +
         document.location.host +
         "/log_streaming_ws");
```